Web/stockdogweb/src/components/Trade/Transaction/Transaction.js

```diff
@@ -0,0 +1,92 @@
+import React, { Component } from 'react';
+import Modal from 'react-responsive-modal';
+import Select from 'react-select';
+import Button from '../../Button/Button';
+import './Transaction.css';
+
+const selectOptions = [
```

Comment:
This name is kind of confusing because it's really similar to the one in the state. Since it's just used for all of the options and they're not necessarily selected, you could maybe name this 'availableOptions' or something like that.

Reply:
Good idea